### PR TITLE
Removes various mentions of the Syndicate, labels "Nutri-Cakes" as an improper name

### PR DESCRIPTION
**Pull request by Tag114**
Removes multiple in-game mentions of the Syndicate because it is no longer canon.
Also labels "Nutri-Cakes" as an improper name because the PR involves renaming the trash version of it, which is still called "syndi-cakes".

- Renames the syndicate rucksack, satchel, duffel bag and messenger bag to the tactical rucksack, satchel, duffel bag and messenger bag.
- Changes the syndicate/tactical satchel's description from "A satchel in the new age style of a multi-corporate terrorist organisation." to "A stylish satchel in the same hip and trendy color scheme as many pieces of combat equipment."
- Changes the first part of the red space suit replica's description from "A plastic replica of the syndicate space suit, you'll look just like a real murderous syndicate agent in this!" to "A crimson red plastic replica of a space suit."
- Changes the red military belt's description from "A syndicate belt designed to be used by boarding parties. Its style is modeled after the hardsuits they wear." to "A red military belt designed to be used by boarding parties and SWAT teams."
- Changes the build name for the laser KA power converter from Syndicate' (it has a loose quote mark next to its name) to Laser.
- Changes the name the stealth armory dropper uses when it makes radio announcements from "Syndicate Autodrone" to "Tactical Autodrone".
- Edits the uplink description of the firing pin to call it "An unmarked firing pin" rather than "A Syndicate-branded Firing pin".
- Edits the uplink description of the encrypted radio channel key to say it gives access to a "secure private frequency" rather than a "secure syndicate frequency".
- Changes the name listed in the uplink for the black and red lunchbox from "Syndicate Lunchbox" to "Tactical Lunchbox".
- Changes the names supply consoles use for the red space suit and helmet and their subtypes when you export them from "Syndicate space suit" and "Syndicate space helmet" to "armored space suit" and "armored space helmet".
- Renames the trash item version of Nutri-Cakes from "syndi cakes" to "Nutri-Cakes".
- Labels the name of the food item version of Nutri-Cakes as improper.

The changelog just says "Removed multiple mentions of the Syndicate." and "The game no longer treats 'Nutri-Cakes' as a proper name." instead of listing every change individually because I think it would be unnecessarily long if it did.

### CODE REVIEW

**Comment by Tag114** [comment]
!review

**Review by DreamySkrell — APPROVED** [review]

File: code/game/objects/items/devices/drop_targeter/orbital_drops.dm [hunk 1]
```diff
@@ -57,7 +57,6 @@
 
 /obj/item/device/orbital_dropper/armory/syndicate
 	desc_antag = "This is a stealthy variant of the standard armory orbital drop. It will not report itself dropping on common, unless emagged."
-	announcer_name = "Syndicate Autodrone"
```
Comment (SleepyGemmy) [inline code/game/objects/items/devices/drop_targeter/orbital_drops.dm:60]: change this to "Tactical Autodrone".

File: code/game/objects/items/trash.dm [hunk 2]
```diff
@@ -60,7 +60,7 @@
 	icon_state = "sosjerky"
 
 /obj/item/trash/syndi_cakes
-	name = "syndi cakes"
+	name = "Nutri-Cakes"
```
Comment (SleepyGemmy) [inline code/game/objects/items/trash.dm:63]:
```suggestion
	name = "\improper Nutri-Cakes"
```

Comment (Tag114) [reply code/game/objects/items/trash.dm]: The food version of Nutri-Cakes isn't labeled as improper either. Is it alright to change that in this PR or should I make a seperate PR for it?

Comment (SleepyGemmy) [reply code/game/objects/items/trash.dm]: you can change it alongside.

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**